internal/featureflags/temporary.go

```diff
@@ -46,6 +46,9 @@ const (
 	// AccessControlSettings toggles whether settings enable the access control
 	// Introduced: v2.21.0
 	AccessControlSettings FeatureFlag = "MONACO_FEAT_ACCESS_CONTROL_SETTINGS"
+	// SanitizeBucketNames toggles whether bucket names created by Monaco are sanitized or not.
+	// Introduced: v2.23.0
+	SanitizeBucketNames FeatureFlag = "MONACO_FEAT_SANITIZE_BUCKET_NAMES"
```

Comment:
Nit, but we don't need to add `FEAT` to everything, as it's not really a feature but a bugfix.

Reply:
I've changed this to `MONACO_SANITIZE_BUCKET_NAMES` - less typing 👍